libs/template/templates/default-python/template/{{.project_name}}/resources/{{.project_name}}_pipeline.yml.tmpl

```diff
@@ -3,6 +3,11 @@ resources:
   pipelines:
     {{.project_name}}_pipeline:
       name: {{.project_name}}_pipeline
+      {{- if eq default_catalog ""}}
+      # catalog: catalog_name
```

Comment:
This should include an explanation about why it is commented on or should be omitted.

Perhaps something like "specify the catalog field to configure this pipeline to write to Unity Catalog".

Reply:
How is

```suggestion
      ## Specify the 'catalog' field to configure this pipeline to make use of Unity Catalog:
      # catalog: catalog_name
```

Reply:
Yeah, that works!